open_gpt/cli/commands/deploy.py

```diff
@@ -30,6 +30,12 @@ class DeployCommand(Command):
             flag=False,
             default='balanced',
         ),
+        option(
+            'backend',
+            None,
+            'The backend of the model.',
+            default='hf',
+        ),
```

Comment:
I would suggest opengpt could decide which backend to use in the runtime. The principle of opengpt is to obtain the best pratice of llm serving out of the box.

Reply:
`vllm` doesn't support streaming output, so if this the default option, it will break the whole user experience.

Reply:
@jemmyshin If this is the case, we should hold the vllm integration.

Reply:
vllm is not the good backend if it does not have stream supporting.